Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
```

Comment:
Please add a `MAINTAINER` clause here.

Reply:
> 
> 
> Please remove the intermittent spaces.
> 
> Thank you for making the pull request. If you are interested to continue working with us, please feel free to join https://app.element.io/#/room/#fedora-websites:matrix.org and drop your introduction.

Sure, I would like to work, but I am new to open source and doing this just as a side gig right now. I will join it and try to contribute further. Also can you please let me know what are intermittent spaces?

Reply:
Is this the right format for adding it?

```
"""
##########################################################################
*
*   Copyright © 2019-2021 <Contributer Name> <Contributer Email>
*
*   This program is free software: you can redistribute it and/or modify
*   it under the terms of the GNU General Public License as published by
*   the Free Software Foundation, either version 3 of the License, or
*   (at your option) any later version.
*
*   This program is distributed in the hope that it will be useful,
*   but WITHOUT ANY WARRANTY; without even the implied warranty of
*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*   GNU General Public License for more details.
*
*   You should have received a copy of the GNU General Public License
*   along with this program.  If not, see <https://www.gnu.org/licenses/>.
*
##########################################################################
"""
```

Reply:
Here's an example https://gist.github.com/t0xic0der/7a3459b272478cf1792c1231fc0d0a03#file-gitblit-noarch-dockerfile-L2

Reply:
Thanks, I will add it